src/update.c

```diff
@@ -190,7 +190,7 @@ static int update_loop(struct list *updates, struct manifest *server_manifest)
 	return ret;
 }
 
-int add_included_manifests(struct manifest *mom, struct list **subs)
```

Comment:
So you updated the function but not all the callers here, I assume this patch didn't even compile.